src/Schema/Table.php

```diff
@@ -56,6 +57,49 @@ public function primary(string $columns, $name = null): ?Blueprint
         return $this->table->setPrimaryKey($columns, $name);
     }
 
+    /**
+     * Dropping an index which satisfy the criteria
+     *
+     * @param string|array $name
+     * @param Closure $satisfy
+     */
+    protected function _dropIndex($name, $satisfy){
```

Comment:
What is the usecase for $satisfy? If you know the name of the index you want to drop, why would you not know the type?

Reply:
@eigan If there fulltext index and unique index for same columns, Then `->dropIndex(['column1','column2'])` will dropping the both indexes. With the `$satisfy` user can specify the type of index.

Reply:
Anyway I can adjust this PR if it is not a common use case.

Reply:
Since neither Doctrine nor Laravel have this possibility then I don't see why we should either. Cool idea though, just don't want to add too much custom that we need to maintain :)

Reply:
@eigan I removed the `$satisfy` as you mentioned.